src/plugins/enzymes/src/FindEnzymesDialog.ui

```diff
@@ -113,38 +113,137 @@
     </widget>
    </item>
    <item>
-    <widget class="QGroupBox" name="groupBox_2">
+    <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
-      <string>Show only selected suppliers:</string>
+      <string>Show/hide enzymes:</string>
```

Comment:
nit: Use "Filter" label

Reply:
I tried to use the word "Show" when we are talking about visibility of enzymes, which you are capable to choose. "Filter" is already in use in parameters, which describes the number of enzymes, which will be found (during the searching process). I thought, that it allows users to slit these two types of parameters:
![image](https://github.com/ugeneunipro/ugene/assets/26257527/33edebde-af10-42db-87dc-328dfc61abd8)

I could change it, if you want.